lib/skate_web/channels/detours_channel.ex

```diff
@@ -0,0 +1,62 @@
+defmodule SkateWeb.DetoursChannel do
+  @moduledoc false
+
+  use SkateWeb, :channel
+  use SkateWeb.AuthenticatedChannel
+
+  alias Skate.Detours.Detours
+
+  # Active
+  @impl SkateWeb.AuthenticatedChannel
+  def join_authenticated("detours:active", _message, socket) do
+    SkateWeb.Endpoint.subscribe("detours:active")
+    %{id: user_id} = Guardian.Phoenix.Socket.current_resource(socket)
+    detours = Detours.grouped_detours(user_id)[:active]
+    {:ok, %{data: detours}, socket}
+  end
+
+  @impl SkateWeb.AuthenticatedChannel
+  def join_authenticated("detours:active" <> route_id, _message, socket) do
+    SkateWeb.Endpoint.subscribe("detours:active" <> route_id)
```

Comment:
I _think_ this is better? otherwise I think this would match on things like `detours:activejoafjoifwaiojfewijowef` where `route_id` would contain `joafjoifwaiojfewijowef`
```suggestion
  def join_authenticated("detours:active:" <> route_id, _message, socket) do
    SkateWeb.Endpoint.subscribe("detours:active:" <> route_id)
```

Reply:
Absolutely!! This was a mistake, and in fact, the frontend expects the colon.

Reply:
I believe it still would've worked, `route_id` would just be `:66` instead of `66`.